ecs-cli/modules/commands/flags.go

```diff
@@ -33,18 +33,19 @@ const (
 	CFNStackNamePrefixDefaultValue       = "amazon-ecs-cli-setup-"
 
 	// Cluster
-	AsgMaxSizeFlag    = "size"
-	VpcAzFlag         = "azs"
-	SecurityGroupFlag = "security-group"
-	SourceCidrFlag    = "cidr"
-	EcsPortFlag       = "port"
-	SubnetIdsFlag     = "subnets"
-	VpcIdFlag         = "vpc"
-	InstanceTypeFlag  = "instance-type"
-	ImageIdFlag       = "image-id"
-	KeypairNameFlag   = "keypair"
-	CapabilityIAMFlag = "capability-iam"
-	ForceFlag         = "force"
+	AsgMaxSizeFlag              = "size"
+	VpcAzFlag                   = "azs"
+	SecurityGroupFlag           = "security-group"
+	SourceCidrFlag              = "cidr"
+	EcsPortFlag                 = "port"
+	SubnetIdsFlag               = "subnets"
+	VpcIdFlag                   = "vpc"
+	InstanceTypeFlag            = "instance-type"
+	ImageIdFlag                 = "image-id"
+	KeypairNameFlag             = "keypair"
+	CapabilityIAMFlag           = "capability-iam"
+	NoAutoAssignPublicIPAddress = "no-associate-public-ip-address"
```

Comment:
Nitpick: lets keep the name same as value and end the name with "Flag" suffix. So "NoAssociatePublicIPAddressFlag"